### ISSUE TITLE
A bigger icon will better on the Microsoft Store

### ISSUE THREAD

**Issue by hidp123** [Simizfo/WSATools#17]
Currently, the icon looks a bit small on the Microsoft Store, a bigger one would be better. See screenshot of some other apps for comaprison.
![image](https://user-images.githubusercontent.com/84683703/147849718-c4d044c4-aa4f-471b-a451-1ff5ef213a21.png)
![image](https://user-images.githubusercontent.com/84683703/147849735-44066d58-41c1-4a7d-ac87-dd9d34072853.png)

**Comment by Simizfo** [comment]
Hmmmm, yeah, thanks! I didn't notice it, but you're definitely right.

This is not something I'll work on right now, sorry for that, but I definitely will at some point. Maybe I'll also change the icon, not sure I like the current one which was made in a hurry to publish the app.

**Comment by MarcAnt01** [comment]
> Hmmmm, yeah, thanks! I didn't notice it, but you're definitely right.
> 
> This is not something I'll work on right now, sorry for that, but I definitely will at some point. Maybe I'll also change the icon, not sure I like the current one which was made in a hurry to publish the app.

I agree, the current icon is too much W-10 like. I'd rather have a flat one instead
